builder/builder-next/builder.go

```diff
@@ -624,6 +632,7 @@ func toBuildkitUlimits(inp []*container.Ulimit) (string, error) {
 	return strings.Join(ulimits, ","), nil
 }
 
+// FIXME(thaJeztah): wire-up new fields; see https://github.com/moby/moby/issues/48639
```

Comment:
I decided to keep the daemon-config as it was, so only made the changes internal for now, but added a bunch of TODO's / FIXME's, and a ticket to look at what's needed to implement the new options and deprecate the old ones;

- https://github.com/moby/moby/issues/48639